plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/itemlist/ObjectListControl.java

```diff
@@ -1479,6 +1479,33 @@ private String copyGridToText() {
         @Override
         public void fillConfigMenu(IContributionManager menuManager) {
             super.fillConfigMenu(menuManager);
+            // Traverse all columns, if there is a column with name "#", add a
+            // menu option "Revert sorting"
+            // which is to sort in the ascending order of the "#" column
+            int columnsCount = this.getColumnsCount();
+            int numberColumnInd = -1;
+            for (int i = 0; i < columnsCount; i++) {
+                if (this.getColumnName(i).equals("#")) {
```

Comment:
Could you clarify why are you searching for a column with name # here?

Reply:
Because the original order of the rows is just the ascending order of the "#" column.